tests/content/util.js

```diff
@@ -244,6 +244,18 @@ const PROCESS_YT_VIDEO_TITLE_DATA = [{
 	description: 'should remove 【genre】 from the beginning of the title',
 	args: ['【Genre】 Artist - Track'],
 	expected: { artist: 'Artist', track: 'Track' },
+}, {
+	description: 'should remove CD track number from the beginning of the title',
+	args: [' 01. Artist - Track'],
+	expected: { artist: 'Artist', track: 'Track' },
+}, {
+	description: 'should remove vinyl track number from the beginning of the title',
+	args: [' A1. Artist - Track'],
+	expected: { artist: 'Artist', track: 'Track' },
+}, {
+	description: 'should remove complex vinyl track number from the beginning of the title',
+	args: [' AA1. Artist - Track'],
+	expected: { artist: 'Artist', track: 'Track' },
```

Comment:
Should we extend this with the ones that should be left intact? e.g. some of the test cases in the regex101 URL? I feel like this test suite should get a bit more extensive.

Reply:
I can add more tests, is it fine if I add them here or should I create a new function in the code as well as a new test function?

Reply:
Added more tests